sql-plugin/src/main/340+/scala/com/nvidia/spark/rapids/shims/CreateDataSourceTableAsSelectCommandMetaShim.scala

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
The file name should be CreateDataSourceTableAsSelectCommandMetaShim**s**, missed the trailing `s`

Reply:
Thank you! Fixed